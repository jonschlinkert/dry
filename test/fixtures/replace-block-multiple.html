{% extends "multiple" %}
{% block replace="header" %}
header: "replace-block-multiple.html"
{% endblock %}
{% block replace="body" %}
body: "replace-block-multiple.html"
{% endblock %}
{% block replace="footer" %}
footer: "replace-block-multiple.html"
{% endblock %}