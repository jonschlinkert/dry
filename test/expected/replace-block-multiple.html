
head: "blocks/multiple.html"

header: "replace-block-multiple.html"

body: "replace-block-multiple.html"

footer: "replace-block-multiple.html"

foot: "blocks/multiple.html"
